src/main/kotlin/uk/gov/justice/digital/hmpps/bodyscan/apiclient/BodyScanApiClient.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.bodyscan.apiclient
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.bodyscan.apiclient.model.PersonalCareCounter
+import uk.gov.justice.digital.hmpps.config.typeReference
+import java.time.LocalDate
+import java.time.format.DateTimeFormatter
+
+@Component
+class BodyScanApiClient(@Qualifier("prisonApiWebClient") private val webClient: WebClient) {
```

Comment:
we usually name clients over the service they call, so lets call this `BodyScanPrisonApiClient` - this codebase will be the bodyScan service so its a bit weird to call the client bodyscanclient

Reply:
sorted